package.json

```diff
@@ -23,8 +23,8 @@
     "clear": "lerna run clear",
     "dist": "yarn clear && lerna run dist",
     "fix": "yarn fix:docs && yarn fix:config && yarn fix:style && yarn fix:script",
-    "fix:config": "prettier --write **/*.json",
-    "fix:docs": "prettier --ignore-path .gitignore --write **/*.md",
+    "fix:config": "prettier --ignore-path .prettierignore --write **/*.json",
+    "fix:docs": "prettier --ignore-path .prettierignore --write **/*.md",
```

Comment:
also `.prettierignore` is ignored by default.. so you don't need to specifically say it

Reply:
https://prettier.io/docs/en/ignore.html

Reply:
also instead of `fix:docs, fix:config, fix:style` we can have one:

```
"fix:assets": "prettier --write **/*.{json,md,scss}",
```